rclpy/rclpy/logging_service.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2023 Sony Group Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from rcl_interfaces.msg import LoggerLevel, SetLoggerLevelsResult
+from rcl_interfaces.srv import GetLoggerLevels
+from rcl_interfaces.srv import SetLoggerLevels
+import rclpy
+from rclpy.impl.logging_severity import LoggingSeverity
+from rclpy.qos import qos_profile_services_default
+from rclpy.validate_topic_name import TOPIC_SEPARATOR_STRING
+
+ERR_MSG_INVAILD_LOGGER_NAME = 'Logger name is invaild.'
+ERR_MSG_INVAILD_LOGGER_LEVEL = 'Logger level is invaild.'
+ERR_MSG_LOGGING_INTERNAL_ERROR = 'Logging system internal error.'
+
+
+class LoggingService:
+
+    def __init__(self, node):
+        node_name = node.get_name()
+
+        get_logger_name_service_name = \
+            TOPIC_SEPARATOR_STRING.join((node_name, 'get_logger_levels'))
+        node.create_service(
+            GetLoggerLevels, get_logger_name_service_name,
+            self.__get_logger_levels, qos_profile=qos_profile_services_default
+        )
+
+        set_logger_name_service_name = \
+            TOPIC_SEPARATOR_STRING.join((node_name, 'set_logger_levels'))
+        node.create_service(
+            SetLoggerLevels, set_logger_name_service_name,
+            self.__set_logger_levels, qos_profile=qos_profile_services_default
+        )
+
+    def __get_logger_levels(self, request: GetLoggerLevels.Request,
+                            response: GetLoggerLevels.Response):
+        for name in request.names:
+            level = LoggerLevel()
```

Comment:
since `level.level` is used below, how about using `logger_level` instead of `level` here.